### DISCUSSION TITLE
FixedBy/Fixed in Version showing '0:0'

### DISCUSSION THREAD

**Discussion by uchi-mata** [quay/quay#1503] [Q&A]
Hi,

I see a lot of vulnerability reports indicate that a vulnerability is 'Fixed in Version: 0:0' (json: "FixedBy": "0:0"). Random example [here](https://quay.io/repository/astronomer/ap-airflow/manifest/sha256:3fd39b3a22900a1c1e05ce7096b1459f17c4c21d266396598387615cd8b9c428?tab=vulnerabilities) but this is the case in many images. 

Does anybody know what that means? Is it a bug? I could not find an answer in the docs or with a cursory code search. 

Thank you, 
Matthias

**Comment by crozzy** [comment]
Hi @uchi-mata, [Clair](https://github.com/quay/clair) is doing the security analysis for Quay and the `0:0` fixed in version has come up before. It is due to the Debian OVAL feel reporting this version and there being no definitive answer as to what it means (from what we've seen it seems to encompass "not affected" and "will not fix" states, which considering Debian's support policy should mean unsupported distros slowly accrue more vulnerabilities that have `0:0` as a fixed version). 

Here is a little context https://github.com/quay/clair/issues/1270, feel free to move the discussion over to Clair if you don't find the explanation you need.

**Comment by uchi-mata** [comment]
That is very helpful, thank you very much!
